libparsers/tests/xml/dynamic_lighting.cpp

```diff
@@ -0,0 +1,93 @@
+#include "parsers/xml/dynamic_lighting.hpp"
+#include <doctest.h>
+#include <iostream>
+
+using namespace parsers::xml;
+using dynamic_lighting::lighting_info;
+using parsers::errors::error_t;
+using parsers::errors::errors_t;
+
+TEST_SUITE_BEGIN("libparsers - xml - dynamic_lighting");
+
+TEST_CASE("libparsers - xml - dynamic_lighting - test") {
+	std::string const test_value =
```

Comment:
turn clang format off with `// clang-format off` before this line, then indent the xml as you normally would, then turn it on after the xml is over with `// clang-format on`. See https://github.com/cwfitzgerald/openbve2/blob/feature-libparsers-xml-dynamicbackground-test/libparsers/tests/xml/dynamic_background.cpp#L14